pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -73,6 +73,11 @@ def model_to_device(self) -> None:
     def is_global_zero(self) -> bool:
         """Whether the current process is the rank zero process not only on the local node, but for all nodes."""
 
+    @property
+    @abstractmethod
```

Comment:
Maybe don't have it as `abstractmethod`, and keep the default as False

Reply:
It is safer to make it abstract.